src/lib/cache/memory/memory.go

```diff
@@ -50,7 +50,7 @@ func (c *Cache) Contains(ctx context.Context, key string) bool {
 	}
 
 	if e.(*entry).isExpirated() {
-		c.Delete(ctx, c.opts.Key(key))
+		_ = c.Delete(ctx, c.opts.Key(key))
```

Comment:
Log error?

Reply:
this is under /src/lib/ directory, in my honest opinion, logging error in lib files may not sound like a good idea. open to discussion though.